supporting_content/columns/servicesubcategory.md

```diff
@@ -0,0 +1,210 @@
+| Storage                   | File Storage                      | EFS                                                                       | NetApp Files                                                                                                                       | Hyperdisk                                                          |
+| Storage                   | Object Storage                    | S3, Glacier                                                               | ADLS                                                                                                                               | Cloud Storage                                                      |
+| Storage                   | Storage Platforms                 |                                                                           | Storage Accounts, HPC Cache, StoreSimple                                                                                           |                                                                    |
+| Storage                   | Other (Storage)                   |                                                                           |                                                                                                                                    |                                                                    |
+| Web                       | Application Frameworks            |                                                                           | App Configuration, Maps, SignalR Service, Fluid Relay                                                                              | Maps {Directions API, Distance Matrix API, Geolocation API, etc.}  |
+| Web                       | Web Platforms                     | Elastic Beanstalk                                                         | App Service, Spring Apps                                                                                                           | App Engine                                                         |
+| Web                       | Other (Web)                       | Lightsail                                                                 |                                                                                                                                    |                                                                    |
+| Other                     | Other (Other)                     |                                                                           |                                                                                                                                    |
+
+### Principles
```

Comment:
nit: We should move the principles above the examples so people see them before they start judging the subcategories 😜

Reply:
Done!